docs/node/RPC Category/16-loan.md

```diff
@@ -350,6 +350,26 @@ interface UTXO {
 }
 ```
 
+## updateVault
+
+Update a vault transaction.
```

Comment:
hmm, seems incomplete. can not update a vault(createvault) transaction.
maybe `Update vault` or `Create update vault transaction `